test/e2e_node/hugepages_test.go

```diff
@@ -85,6 +86,15 @@ func makePodToVerifyHugePages(baseName string, hugePagesLimit resource.Quantity)
 
 // configureHugePages attempts to allocate 10Mi of 2Mi hugepages for testing purposes
 func configureHugePages() error {
+	// Compact memory to make bigger contiguous blocks of memory available
```

Comment:
Nit: If we wanted, we could extract this memoryCompaction to a helper function... up to you :)

Reply:
:+1: 
I think ill leave it like this to make it easier to read. :smile: